src/components/organisms/TimingSpace/TimingSpace.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback, useMemo } from "react";
+
+import { VenueEvent } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { useShowHide } from "hooks/useShowHide";
+
+import { TimingEvent } from "components/organisms/TimingEvent";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./TimingSpace.scss";
+
+export type TimingSpaceProps = {
+  spaceName: string;
+  spaceEvents: WithId<VenueEvent>[];
+  setEditedEvent: React.Dispatch<
+    React.SetStateAction<WithId<VenueEvent> | undefined>
+  >;
+  setShowCreateEventModal: () => void;
+};
+
+export const TimingSpace: React.FC<TimingSpaceProps> = ({
+  spaceName,
+  spaceEvents,
+  setShowCreateEventModal,
+  setEditedEvent,
+}) => {
+  const { isShown: selectedSpace, toggle: toggleSelectedSpace } = useShowHide();
+
+  const handleCreateButtonClick = useCallback(
+    (e) => {
+      e.stopPropagation();
+
+      setShowCreateEventModal();
+    },
+    [setShowCreateEventModal]
+  );
+
+  const renderedSpaceEvents = useMemo(
+    () =>
+      spaceEvents?.map((event) => (
+        <TimingEvent
+          event={event}
+          setShowCreateEventModal={setShowCreateEventModal}
+          setEditedEvent={setEditedEvent}
+          key={event.id}
+        />
+      )),
+    [spaceEvents, setShowCreateEventModal, setEditedEvent]
+  );
+
+  const spaceNameTitle = useMemo(() => {
+    if (!spaceEvents.length) {
+      return "nothing scheduled";
+    }
+
+    if (spaceEvents.length === 1) {
+      return "1 experience";
+    }
+
+    return `${spaceEvents.length} experiences`;
+  }, [spaceEvents.length]);
+
+  return (
+    <div className="TimingSpace">
+      <div className="TimingSpace__header" onClick={toggleSelectedSpace}>
+        <span>
+          <span className="TimingSpace__name">{spaceName}</span>
+          <span className="TimingSpace__title">{spaceNameTitle}</span>
+        </span>
+        <ButtonNG
+          className="TimingSpace__createButton"
+          onClick={handleCreateButtonClick}
+        >
+          Add experience
```

Comment:
This button says Add experience but the other one says, Create an Event. We need to unify them, please ask Sofi or product about this.

Reply:
@sofisparkle, please clarify Do we need to rename "create an event" into "add experience" at all? 
Is it the correct way to update functionality?

Reply:
@yurii-lubynets You should ping her in slack she probably won't see this.

Reply:
Yes change to "Add Experience"

Reply:
@denisdimitrov fixed in 451ce9bf8a59a3465d5ae12b2bdb8dd01428a06a